src/source/PeerConnection/Rtcp.c

```diff
@@ -60,7 +60,6 @@ static STATUS onRtcpSenderReport(PRtcpPacket pRtcpPacket, PKvsPeerConnection pKv
 
     if (pRtcpPacket->payloadLength != RTCP_PACKET_SENDER_REPORT_MINLEN) {
         // TODO: handle sender report containing receiver report blocks
-        DLOGW("unhandled packet type RTCP_PACKET_SENDER_REPORT size %d", pRtcpPacket->payloadLength);
```

Comment:
Recommend changing this to DLOGS. These won't compile unless LOG_STREAMING compiler define is used.

Reply:
Agreed. But, when compiling with `LOG_STREAMING` define, it still produces a spew. Which is not required at the time since we know multiple report blocks are not supported at the moment.

Reply:
I would have preferred to keep it but at a lower level like INFO. WARN is clearly wrong since it looks like something went wrong when really nothing did.

Reply:
We can keep the logging, but it does not give anything useful to the customers, not even as INFO. So, I would prefer getting rid of it altogether.